Hi @thinkyhead 
**I can't compile it.**

```
Compiling .pio\build\BIGTREE_GTR_V1_0\src\src\lcd\menu\menu_mixer.cpp.o
In file included from Marlin\src\lcd\menu\menu_configuration.cpp:31:
Marlin\src\lcd\menu\menu_configuration.cpp: In function 'void menu_idex()':
Marlin\src\lcd\menu\menu_item.h:259:40: error: no matching function for call to 'TMenuEditItem<MenuEditItemInfo_float42_52>::action(const char* const&, float*)'      
  259 |     MenuItem_##TYPE::action(plabel, ##V);                      \
      |                                        ^
Marlin\src\lcd\menu\menu_item.h:270:5: note: in expansion of macro '_MENU_INNER_P'
  270 |     _MENU_INNER_P(TYPE, ##V);         \
      |     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_item.h:354:57: note: in expansion of macro '_MENU_ITEM_P'
  354 | #define MENU_ITEM_P(TYPE, PLABEL, V...)                 _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
      |                                                         ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu_item.h:392:58: note: in expansion of macro 'MENU_ITEM_P'
  392 | #define EDIT_ITEM_P(TYPE, PLABEL, V...)                  MENU_ITEM_P(TYPE, PLABEL, ##V)
      |                                                          ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_item.h:393:58: note: in expansion of macro 'EDIT_ITEM_P'
  393 | #define EDIT_ITEM(TYPE, LABEL, V...)                     EDIT_ITEM_P(TYPE, GET_TEXT(LABEL), ##V)
      |                                                          ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_configuration.cpp:207:5: note: in expansion of macro 'EDIT_ITEM'
  207 |     EDIT_ITEM(float42_52, MSG_IDEX_DUPE_GAP, &duplicate_extruder_x_offset);
      |     ^~~~~~~~~
Marlin\src\lcd\menu\menu_item.h:93:17: note: candidate: 'static void TMenuEditItem<NAME>::action(const char*, TMenuEditItem<NAME>::type_t*, TMenuEditItem<NAME>::type_t, TMenuEditItem<NAME>::type_t, screenFunc_t, bool) [with NAME = MenuEditItemInfo_float42_52; TMenuEditItem<NAME>::type_t = float; screenFunc_t = void (*)()]'        
   93 |     static void action(
      |                 ^~~~~~
Marlin\src\lcd\menu\menu_item.h:93:17: note:   candidate expects 6 arguments, 2 provided
*** [.pio\build\BIGTREE_GTR_V1_0\src\src\lcd\menu\menu_configuration.cpp.o] Error 1
```

**if it disables** this line in Marlin \ src \ lcd \ menu \ menu_configuration.cpp

`// EDIT_ITEM (float42_52, MSG_IDEX_DUPE_GAP, & duplicate_extruder_x_offset);`

**FW can be compiled**

--- a/Marlin/src/lcd/language/language_en.h
+++ b/Marlin/src/lcd/language/language_en.h
@@ -144,7 +144,7 @@ namespace Language_en {
   PROGMEM Language_Str MSG_IDEX_MODE_DUPLICATE             = _UxGT("Duplication");
   PROGMEM Language_Str MSG_IDEX_MODE_MIRRORED_COPY         = _UxGT("Mirrored Copy");
   PROGMEM Language_Str MSG_IDEX_MODE_FULL_CTRL             = _UxGT("Full Control");
-  PROGMEM Language_Str MSG_IDEX_DUPE_GAP                   = _UxGT("Duplication Gap");
+  PROGMEM Language_Str MSG_IDEX_DUPE_GAP                   = _UxGT("Duplicate X-Gap");
   PROGMEM Language_Str MSG_HOTEND_OFFSET_X                 = _UxGT("2nd Nozzle X");
   PROGMEM Language_Str MSG_HOTEND_OFFSET_Y                 = _UxGT("2nd Nozzle Y");
   PROGMEM Language_Str MSG_HOTEND_OFFSET_Z                 = _UxGT("2nd Nozzle Z");

--- a/Marlin/src/lcd/menu/menu_configuration.cpp
+++ b/Marlin/src/lcd/menu/menu_configuration.cpp
@@ -204,7 +204,8 @@ void menu_advanced_settings();
       : PSTR("M605S1\nT0\nM605S2\nG28 X\nG1X0\nM605S3")
     );
     GCODES_ITEM(MSG_IDEX_MODE_FULL_CTRL, PSTR("M605S0\nG28X"));
-    EDIT_ITEM(float42_52, MSG_IDEX_DUPE_GAP, &duplicate_extruder_x_offset);
+
+    EDIT_ITEM(float42_52, MSG_IDEX_DUPE_GAP, &duplicate_extruder_x_offset, (X2_MIN_POS) - (X1_MIN_POS), (X_BED_SIZE) - 20);
 
     END_MENU();
   }

--- a/buildroot/tests/mega2560-tests
+++ b/buildroot/tests/mega2560-tests
@@ -259,10 +259,10 @@ exec_test $1 $2 "Mightyboard Rev. E | CoreXY, Gradient Mix | Endstop Int. | Home
 
 ######## Example Configurations ##############
 #
-# Test a full-featured CR-10S config
+# Test a basic DUAL_X_CARRIAGE configuration
 #
-use_example_configs Creality/CR-10S
-exec_test $1 $2 "Full-featured CR-10S config"
+use_example_configs Formbot/T_Rex_3
+exec_test $1 $2 "Formbot/T_Rex_3 example configuration."
 
 #
 # BQ Hephestos 2

